macosx/Snes9x/AppDelegate.h

```diff
@@ -20,10 +20,17 @@
 
 #import <Cocoa/Cocoa.h>
 #import <snes9x_framework/snes9x_framework.h>
+#import "S9xPreferences/S9xPreferencesWindowController.h"
+
+extern NSWindowFrameAutosaveName const kMainWindowIdentifier;
 
 @interface AppDelegate : NSObject <NSApplicationDelegate, S9xInputDelegate>
 
 @property (nonatomic, readonly, assign) BOOL isRunningEmulation;
+@property (nonatomic, strong) S9xEngine *s9xEngine;
```

Comment:
I won't reject the change for this, but I'll probably move these back to the .m file. The purpose of defining properties in the .m file is for visibility. for properties, Objective-C doesn't have any notion of public/private/protected. Defining them in the .m file makes them private.

Reply:
Please do what you think is best. I shouldn't have moved them around anyways. I think I was just moving some variables around to get an idea of how Objective-C works and whether declaring properties in the .h or .m file would break the compiler.

Reply:
Yeah, no worries on that front. It certainly seemed like something like that was going on, and I just wanted to provide some context that isn't often called out in Objective-C documentation.